src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: flex;
+  justify-content: center;
+
+  &__cards {
+    display: flex;
+    flex-wrap: wrap;
+    padding: $catalog_cards-paddings-vertical 0;
+    margin-bottom: 72 - $catalog_cards-paddings-vertical;
+    width: $card-width * $cards-amount
+      + ($cards-amount - 1) * $cards-margin-between-horizontal
```

Comment:
🚀
also, you can create variable cards-width

Reply:
![image](https://user-images.githubusercontent.com/60798246/180047978-38db60c3-3f34-4f74-841e-1af6e66a0f6c.png)
i was trying to do something like that, if i understood you correctly, but i didn't have time to figure out how to count how many cards are on the page to determine how many of them are the last ones, cos last row should not have margin.